introduction.md

```diff
@@ -1,2 +1,10 @@
 hello its me new student in codeschool mastering github :D
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?